### DISCUSSION TITLE
Manual installation

### DISCUSSION THREAD

**Discussion by lgLindstrom** [hassio-addons/addon-vscode#369] [Q&A] [answered]
I am running HA in Docker together with some other services.   Is it possible to add this addon as a service in my docker-compose file.?? I have tried to find instructions for this but have not been successful. Can you help ?

**Comment by frenck** [comment, marked as answer]
Add-ons are only available in Home Assistant OS or Home Assistant Supervised installation types.

**Reply by lgLindstrom** [reply]
As I understand this "addon" creates a docker image. 
Can I use that image ? 
Where is it deployed?

**Reply by frenck** [reply]
> Can I use that image ?

No, see my previous answer.

**Reply by lgLindstrom** [reply]
ok.
thanks.
